server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -106,6 +106,22 @@ private ConfigProperties() {
      */
     public static final String ACTIVEMQ_MAX_PAGE_SIZE = "candlepin.audit.hornetq.max_page_size";
 
+    /**
+     * For more on Artemis redelivery delays, see:
+     * https://activemq.apache.org/artemis/docs/2.4.0/undelivered-messages.html
+     */
+
+    // The number of milliseconds to wait before redelivering failed messages to the listeners.
```

Comment:
Let's mark these comments as javadoc `/**` so they'll show up in generated documentation.

Reply:
:+1: